packages/medusa/src/api-v2/admin/regions/validators.ts

```diff
@@ -69,3 +69,21 @@ export class AdminGetRegionsParams extends extendedFindParamsMixin({
   @Type(() => AdminGetRegionsParams)
   $or?: AdminGetRegionsParams[]
 }
+
+export class AdminPostRegionsReq {
+  @IsString()
```

Comment:
Just a clarification, the existing tickets are only for updating a region if I understood correctly, and this object is for creating a new Region. Should I also take care of setting the countries during creation of the region as well?

Reply:
> Should I also take care of setting the countries during creation of the region as well?
That would be great!